app/src/ep.config.json

```diff
@@ -31,11 +31,11 @@
   "siteImagesUrl": "https://elasticpath-demo-images.s3-us-west-2.amazonaws.com/VESTRI_VIRTUAL_V1/%fileName%",
   "enableOfflineMode": false,
   "b2b": {
-    "enable": false,
-    "accounts": false,
+    "enable": true,
+    "accounts": true,
```

Comment:
We'll need to add a config to only show requisition lists in the dashboard view if it's set to true. Similar to how we had for "accounts" previously, we just need one for "req-list" as this is intended to be dark code.